$OpenBSD: patch-setup_py,v 1.1 2021/10/21 08:59:34 sthen Exp $

Provide a minimal setup.py until py-setuptools is updated to allow setup.cfg
only projects.

Index: setup.py
--- setup.py.orig
+++ setup.py
@@ -0,0 +1,2 @@
+from setuptools import setup
+setup(use_scm_version=True)
